eZ/Publish/Core/Persistence/Legacy/Content/Location/Gateway/DoctrineDatabase.php

```diff
@@ -1651,15 +1651,22 @@ private function createNodeQueryBuilder(array $translations = null, bool $useAlw
                 $useAlwaysAvailable
             );
 
-            $queryBuilder->innerJoin(
+            $queryBuilder->leftJoin(
                 't',
                 'ezcontentobject',
                 'c',
-                $expr->andX(
-                    $expr->eq('t.contentobject_id', 'c.id'),
+                $expr->eq('t.contentobject_id', 'c.id')
+            );
+
+            $queryBuilder->andWhere(
```

Comment:
```suggestion
            $queryBuilder->where(
```

Reply:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/2815/commits/e53ed718fedc38be8d0442dece94c61c28c66636